client/pom.xml

```diff
@@ -48,14 +48,24 @@
         <artifactId>resteasy-client</artifactId>
         <version>${resteasy.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.jboss.resteasy</groupId>
+        <artifactId>resteasy-client-api</artifactId>
+        <version>${resteasy.version}</version>
```

Comment:
The version is already managed in the parent pom, we shouldn't also need to do it here.

Reply:
This is dependency management section. I am getting errors if I don't specify version here. Am I missing something trivial here? 🤔

Reply:
You need to add 

      <dependency>
        <groupId>org.jboss.resteasy</groupId>
        <artifactId>resteasy-client-api</artifactId>
        <version>${resteasy.version}</version>
      </dependency>

to the parent pom, then you can remove the resteasy dependencies from here.

Reply:
999b146f